### ISSUE TITLE
Add a link to the API Documentation in the main header

### ISSUE THREAD

**Issue by winslet** [fabric-composer/fabric-composer#101]
To give context:

The original question, regarding the site at: https://fabric-composer.github.io/jsdoc/develop/index.html 
1. Are they our auto-generated docs?
2. Do we still need them?
3. Could we potentially bring that info into the main docs bit of the site?

--

A1. Yes
A2. Yes
A3. Yes, to customise the look and feel we would need to put together a template (currently using this template: https://github.com/docstrap/docstrap ) 

--

**Immediate Action:** Add a link to the API documentation at the top level navigation eg here:

![image](https://cloud.githubusercontent.com/assets/13980028/22696102/cfd67c54-ed44-11e6-9ab1-becd61af3c2f.png)

Potentially with the label "API Documentation"

**Long Term Action:** Look into putting together a template file to modify the experience the build template gives out.

**Comment by mbwhite** [comment]
The develop bit is unrequired.. just removing that at the moment.

**Comment by mbwhite** [comment]
The template that is used for generating the JSDocs is the https://github.com/docstrap/docstrap  'spacelab' theme;  previously that was pulled in from the npm module it was installed via.

However, I've moved (copied actually) this to be in the /fabric-composer/site/jsdoc-template directory.
I've managed to change the navbar header to match the same teal colour.

Looks like it's going be a combination of emerging that CSS with the site's main CSS
